src/lib/api-routes/oauth2.js

```diff
@@ -52,7 +52,6 @@ router.post('/token', async (req, res, next) => {
   try {
     const plainIdToken = {
       username: req.body.username,
-      dfspId: req.body.username,
```

Comment:
Is there any specific reason you want to delete this? I think connection manager is using the dfspId from the oauth token.